test/conftest.py

```diff
@@ -14,7 +14,12 @@
 from snowflake.connector import SnowflakeConnection
 from snowflake.connector.telemetry import TelemetryClient, TelemetryData
 
-from . import CLOUD_PROVIDERS, PUBLIC_SKIP_TAGS, running_on_public_ci
+from . import (
+    CLOUD_PROVIDERS,
+    EXTERNAL_SKIP_TAGS,
```

Comment:
Are these tags going to be necessary if we run the tests against BC sfctest accounts?

Reply:
We need a way to skip running these tests when we run them from Jenkins, the tag `external` means not to run on Jenkins and conversely `internal` tests will only run on Jenkins. We already had `internal`, I just added logic for `external`